docker-compose.yml

```diff
@@ -4,6 +4,7 @@ networks:
   backend:
     driver: overlay
     name: backend
+    attachable: true
```

Comment:
Nice! What was this for? Does it make the `docker exec ...` commands work for restore? Or were you just wanting to be able to do `docker-compose run --entrypoint /bin/bash <service>` on anything on the network?

Reply:
Now that I think about it, this probably allows `docker-compose run database-backup restore` to use `postgres` as the DNS name for the postgres service.

Reply:
I removed this becuase this is unnecessary when using docker swarm. this was necessary when running docker compose but were not doing that anymore.